packages/snippet-bot/src/api-labels.ts

```diff
@@ -28,15 +31,51 @@ export interface ApiLabels {
   products: Array<ApiLabel>;
 }
 
+export const getDriftApiLabels = async (
```

Comment:
Please add a comment for this getDriftApiLabels and mergeApiLabels.

Reply:
Done.